src/framework/preferences/css/sass/PrefsEditor.scss

```diff
@@ -193,12 +193,19 @@ $icon-font: 'PrefsFramework-Icons';
     /* height and width specified in ems instead of rems
        to respond to emphasize links preferences */
     .fl-textfieldStepper .fl-textfieldStepper-textField {
+        -moz-appearance: textfield;
         height: 2em;
         margin: 0 0.5em;
         text-align: center;
         width: 4em;
     }
 
+    .fl-textfieldStepper .fl-textfieldStepper-textField::-webkit-inner-spin-button,
+    .fl-textfieldStepper .fl-textfieldStepper-textField::-webkit-outer-spin-button {
+        appearance: none;
+        margin: 0;
+    }
+
```

Comment:
Shouldn't these new additions already be coming from TextfieldStepper.css?

Reply:
The prefs framework and UIO demos don't include the TextfieldStepper stylesheet— it appears that for some historical reason, those styles were duplicated in the Prefs Framework stylesheet.

Reply:
Hmm... maybe it was to simplify the imports, not sure.